config/environments/production.rb

```diff
@@ -56,6 +56,7 @@
 
   # Use a different cache store in production.
   # config.cache_store = :mem_cache_store
+  config.cache_store = :memory_store
```

Comment:
I think we probably want this to be file store - as I believe this is the normal for GOV.UK apps (and probably has a few advantages)

Reply:
removed